pkgs/development/cuda-modules/setup-hooks/mark-for-cudatoolkit-root-hook.sh

```diff
@@ -1,14 +1,25 @@
 # shellcheck shell=bash
 
-# Should we mimick cc-wrapper's "hygiene"?
-[[ -z ${strictDeps-} ]] || (( "$hostOffset" < 0 )) || return 0
+(( ${hostOffset:?} == -1 && ${targetOffset:?} == 0)) || return 0
```

Comment:
Side note: I believe that "in practice" the derivation never sees offsets other than `{-1, 0, 1}`, but I saw other hooks write `< 0` instead of `== -1`... I really can't say what this change means in the larger scheme of things